templates/github/workflows/CI.yml

```
@@ -1,7 +1,12 @@
 name: CI
 on:
-  - push
-  - pull_request
+  push:
+    <<#BRANCH>>
+    branches:
```

i wonder if we want to swap the order here, so the `branches:` bit is always included?

```suggestion
    branches:
    <<#BRANCH>>
```

---

I think you were right to suggest that. From the docs (which answer the question I posed above):
> If you define only tags or only branches, the workflow won't run for events affecting the undefined Git ref.